ckan-2.10/base/Dockerfile

```diff
@@ -89,7 +89,7 @@ RUN addgroup -g 92 -S ckan && \
 
 # Create local storage folder
 RUN mkdir -p ${CKAN_STORAGE_PATH} && \
-    chown -R ckan:ckan ${CKAN_STORAGE_PATH}
+    chown -R ckan:ckan-sys ${CKAN_STORAGE_PATH}
```

Comment:
The idea is to let both the ckan and ckan-sys users write to this directory? Good idea but it might not work because sub-directories will be owned by the user that created them.

Reply:
No not really, I think the `ckan` user needs to own the `/var/lib/ckan/` directory unless I start changing the group permissions of the directories that potentially both the `ckan` and `ckan-sys` users would write files into...I'd look at updating the umask value for both users if that was the case